.github/workflows/unitest.yaml

```diff
@@ -8,13 +8,17 @@ jobs:
   unitest:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-go@v2
+    - uses: actions/checkout@v4
+    - uses: actions/setup-go@v5
       with:
-        go-version: '1.22.1'
+        go-version: '1.22.9'
```

Comment:
isn't neuvector repo using 1.22.7 ?

Reply:
Yes.  I think we can go faster on unit test given that it’s usually compatible and it can help to discover issues earlier if it’s not the case when we upgrade to 1.22.9 later.